README.md

```diff
@@ -44,23 +45,24 @@ python3 -m pip install --user -U  colcon-common-extensions rosdep vcstool
 We need to setup our sysroot directory for the docker image. Docker can only copy from a specific
 context so all these assets need to be copied relative to the `Dockerfile` path.
 ```bash
-# Create a directory to store qemu assets
-mkdir -p sysroot/qemu-user-static
-cp /usr/bin/qemu-* sysroot/qemu-user-static 
+cd /path/to/sysroot
+# Create a directory to store ROS/qemu assets
+mkdir qemu-user-static ros2_ws
+cp /usr/bin/qemu-* qemu-user-static 
 # Copy ROS Sources
-cp -r ~/ros2_ws/src sysroot
+cp -r ~/ros2_ws/src ros2_ws
```

Comment:
The dockerfile currently expects to find ros2 sources in [<sysroot_path>/src/](https://github.com/aws-ros-dev/cross_compile/blob/cc-sysroot-creator/sysroot_creator/scripts/sysroot/Dockerfile_workspace#L16). Let's keep it consistent with that?